File: CivitasJava/juegoTexto/TestP3.java
package juegoTexto;

import civitas.CivitasJuego;
import civitas.Dado;
import java.util.ArrayList;

public class TestP3 {
	public static void main(String[] argv) {
		ArrayList<String> nombres = new ArrayList<>();
		nombres.add("jugador1");
		nombres.add("jugador2");
		nombres.add("jugador3");
		nombres.add("jugador4");
		
		CivitasJuego juego = new CivitasJuego(nombres);
		VistaTextual vista = new VistaTextual();
		Dado.getInstance().setDebug(true);
		Controlador controlador = new Controlador(juego, vista);
		controlador.juega();
	}
}

File: CivitasRuby/test_p3.rb
require_relative 'civitas_juego'
require_relative 'vista_textual'
require_relative 'dado'
require_relative 'controlador'

module Civitas
  class TestP3
    def initialize
      main
    end
    
    def main
      nombres = ["Jugador1", "Jugador2", "Jugador3", "Jugador4"]
      juego = CivitasJuego.new(nombres)
      vista = Vista_textual.new()
      Dado.instance.set_debug(true)
      controlador = Controlador.new(juego,vista)
      controlador.juega
    end
  end
  
  TestP3.new
end
